src/proto/proto_common.cpp

```diff
@@ -401,6 +401,14 @@ void init_data_readers(
 
     reader->set_master(master);
 
+    //  under development; experimental
+    if (opts->has_bool("use_data_store") && opts->get_bool("use_data_store")) {
```

Comment:
You only need opts->get_bool here; it "use_data_store" isn't in the map, false is returned by default